afs-cassandra/src/test/java/com/powsybl/afs/cassandra/CassandraRenameIssueTest.java

```diff
@@ -6,39 +6,25 @@
  */
 package com.powsybl.afs.cassandra;
 
-import com.powsybl.afs.storage.InMemoryEventsBus;
+import com.powsybl.afs.storage.AppStorage;
 import com.powsybl.afs.storage.NodeGenericMetadata;
 import com.powsybl.afs.storage.NodeInfo;
-import org.cassandraunit.CassandraCQLUnit;
-import org.cassandraunit.dataset.cql.ClassPathCQLDataSet;
-import org.junit.Ignore;
-import org.junit.Rule;
-import org.junit.Test;
 
 import static org.junit.Assert.assertEquals;
 
 /**
  * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
  */
-@Ignore
 public class CassandraRenameIssueTest {
 
-    @Rule
-    public CassandraCQLUnit cassandraCQLUnit = new CassandraCQLUnit(new ClassPathCQLDataSet("afs.cql", CassandraConstants.AFS_KEYSPACE), null, 20000L);
-
-    @Test
-    public void test() {
-        CassandraTestContext context = new CassandraTestContext(cassandraCQLUnit);
-        CassandraAppStorageConfig config = new CassandraAppStorageConfig().setBinaryDataChunkSize(10);
-        CassandraAppStorage storage = new CassandraAppStorage("test", () -> context, config, new InMemoryEventsBus());
-        NodeInfo rootNodeId = storage.createRootNodeIfNotExists("test", "folder");
-        NodeInfo test1NodeInfo = storage.createNode(rootNodeId.getId(), "test1", "folder", "", 0, new NodeGenericMetadata());
-        storage.createNode(test1NodeInfo.getId(), "test2", "folder", "", 0, new NodeGenericMetadata());
-        storage.createNode(test1NodeInfo.getId(), "test2_bis", "folder", "", 0, new NodeGenericMetadata());
+    public void test(AppStorage storage) {
+        NodeInfo rootNodeId = storage.createRootNodeIfNotExists("CassandraRenameIssueTest", "folder");
+        NodeInfo newNode = storage.createNode(rootNodeId.getId(), "CassandraRenameIssueTest_test2", "folder", "", 0, new NodeGenericMetadata());
```

Comment:
I don't know what was the initial issue, bu you have changed the test a little (there was 2 more nodes created and checked).

We should make sure all previous tests still passe without changing them, instead.

I think you have done it in ordre to shorten the execution time, by not re-starting the cassandra server for every test:
maybe in order to achieve this we should drop the keyspace between unit tests, instead of always re-using the previous data.